README.rst

```diff
@@ -21,7 +21,7 @@ Three main areas of functionality are included:
 - General fits file cutouts incuding from single images and sets of images with the shared WCS/pixel scale.
 - Cutout post-processing functionality, including centering cutouts along a path (for moving targets) and combining cutouts.
 
-Documentation is at https://astrocut.readthedocs.io.
+Documentation is at https://astrocut.readthedocs.io. If you'd like to practice working with the tools discussed in this Wiki, we encourage you to work with our [astrocut-for-beginners Jupyter Notebook](https://github.com/spacetelescope/mast_notebooks/tree/main/notebooks/astrocut/making_tess_cubes_and_cutouts), which shows you how to use the tools within make_cube.py and cube_cut.py to generate cutouts from a batch of TESS FFIs retrieved locally via astroquery.mast.
```

Comment:
just curious, why doesn't this show up here? https://spacetelescope.github.io/mast_notebooks/

Reply:
oh, it does, here: https://spacetelescope.github.io/mast_notebooks/notebooks/astrocut/making_tess_cubes_and_cutouts/making_tess_cubes_and_cutouts.html

but "astrocut" doesn't render on the sidebar. is it because there's no `markdown` file in the root of the `astrocut/` directory?

guess i'm just wondering if the link should go to this rendered page instead?

Reply:
For that website with all the rendered notebooks, there's no sidebar nav for astrocut. So people can't discover this page on their own. Is that intentional?

Reply:
A question maybe for @ttdu?

Reply:
I will fix this. RIght now, the sidebar does not automatically update based on the repository content, so I will manually add the astrocut notebook.